.github/workflows/examples.yml

```diff
@@ -19,10 +19,21 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install
+    - name: Setup cache
+      uses: actions/cache@v2
+      env:
+        cache-name: daily-example
```

Comment:
What happens if we don't distinguish by workflow, does it still work?

Assuming we have to, a bit curious whether you could use some of the [default envs](https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables), such as `GITHUB_ACTION`. Not a request but wanted to know if you've tried.

Reply:
Ah, although `setup.py` is identical, different libraries are installed for different workflows.